includes/Core/Authentication/Authentication.php

```diff
@@ -1327,6 +1330,17 @@ private function get_proxy_permissions_url() {
 		);
 	}
 
+	/**
+	 * Gets the proxy support URL.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string Support URL.
```

Comment:
```suggestion
	 * @return string|null Support URL.
```

(assuming the default value is changed)